### DISCUSSION TITLE
Using Vercel Postgres

### DISCUSSION THREAD

**Discussion by taiqn** [dubinc/dub#289] [Announcements]
Now Vercel has its own Postgres, it would be nice if you could instruct how to implement it

Thanks
Tai

**Comment by steven-tey** [comment]
Since Dub uses Prisma, you can just swap out the `DATABASE_URL` env var, change a few of the column formats in the Prisma schema, and it should work!

Closing this for now – hope this helps!

**Comment by Danm72** [comment]
Hey @steven-tey I'd probably keep this open and maybe make that part of a readme/onboarding I think its a preferred flow to keep a lot of the dependencies in Vercel rather than setting up other subs!

**Comment by steven-tey** [comment]
@Danm72 good point, converting it into a discussion instead!

**Comment by w33ts** [comment]
Hey hey! Just a heads up, I did attempt to go down the PostgreSQL route (because I'm cheap and didn't want to pay PlanetScale for another DB), but I ran into this issue here with the @@FullText part of the Prisma Schema: https://github.com/prisma/prisma/issues/8950

Now, I'm sure there is a workaround or a way to handle it in the code, but I'm not that talented. Hopefully that saves you some time if you add compatibility for PostgreSQL!
